conanfile.py

```diff
@@ -29,7 +29,8 @@ class ImageMagicConan(ConanFile):
                "webp": [True, False],
                "xml": [True, False],
                "freetype": [True, False],
-               "utilities": [True, False]}
+               "utilities": [True, False],
+               "with_libjpeg": [True, False, "libjpeg-turbo"]}
```

Comment:
```suggestion
               "with_libjpeg": [False, "libjpeg", "libjpeg-turbo"]}
```

We had a similar discussion on Conan Center Index few days ago [here](https://github.com/conan-io/conan-center-index/pull/2730).